docs/data/base/components/tabs/tabs.md

```diff
@@ -67,6 +67,12 @@ The next demo shows how to apply custom styles to a set of tabs:
 
 {{"demo": "UnstyledTabsCustomized.js"}}
 
+### Vertical
+
+To adapt keyboard navigation (e.g. arrow keys) to use a vertical orientation, set `orientation="vertical"` on the `Tabs` component.
```

Comment:
Let's make this more informative, and also clarify that it's a "must" for a11y purposes.

```suggestion
Tab components can be arranged vertically as well as horizontally.
When vertical, you must set `orientation="vertical"` on the `<Tabs />` component so the user can navigate with the up and down arrow keys (rather than the default left-to-right behavior for horizontal tabs).
```